src/Toolchain.ml

```diff
@@ -0,0 +1,376 @@
+open Bindings
+open Utils
+module P = Js.Promise
+module WorkspaceCfg = Vscode.WorkspaceConfiguration
+
+module Binaries = struct
+  let esy = "esy"
+
+  let opam = "opam"
+end
+
+(* Terminology: - PackageManager - represents supported package managers with
+   Global as the fallback - Manifest - abstracts functions handling manifest
+   files of the supported package managers *)
```

Comment:
Maybe make change `wrap-comments` to `false` in `ocamlformat`? 
Otherwise, it probably should be formatted like: 
```
(* Terminology: *)
(* - PackageManager - represents supported package managers with Global as the
   fallback *)
(* - Manifest - abstracts functions handling manifest files of the supported
   package managers *)
```

Reply:
Setting it to false didn't change this file (it only touched Extension.mli)

Maybe @rgrinberg can help you find what you are looking for?

Reply:
What about using ocamldoc comments: `(** *)` and using literal sections where appropriate.

Reply:
I'm also in favor of turning off `wrap-comments`

Reply:
> Setting it to false didn't change this file (it only touched Extension.mli)

The difference is that `ocamlformat` will not touch line breaks if `wrap-comments` is off. But you have to manually edit comment first.
```ocaml
wrap-comments=true
(* Terminology:
    - PackageManager *)
↓↓↓ ocamlformat ↓↓↓
(* Terminology: - PackageManager *)

wrap-comments=false
(* Terminology:
    - PackageManager *)
↓↓↓ ocamlformat ↓↓↓
(* Terminology:
    - PackageManager *)
```